service/frontend/service.go

```diff
@@ -268,11 +292,25 @@ func (s *Service) Start() {
 	)
 
 	opts, err := s.params.RPCFactory.GetFrontendGRPCServerOptions()
+	kep := keepalive.EnforcementPolicy{
```

Comment:
comment here specifying that the configs below are intended for small scale use case